airflow/api_fastapi/core_api/datamodels/task_instances.py

```diff
@@ -150,3 +152,41 @@ class TaskInstanceHistoryCollectionResponse(BaseModel):
 
     task_instances: list[TaskInstanceHistoryResponse]
     total_entries: int
+
+
+class PatchTaskInstanceBody(BaseModel):
+    """Request body for Clear Task Instances endpoint."""
+
+    dry_run: bool = True
+    new_state: str | None = None
+    note: str | None = None
```

Comment:
https://docs.pydantic.dev/latest/api/types/#pydantic.types.StringConstraints
```suggestion
    note: Annotated[str, StringConstraints(max_length=1000)] | None = None
```

Reply:
Yes that one.

Thanks for you input @Kludex :)

Reply:
Done